examples/deepseek_on_gaudi/single_16k_len.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+
+# set -x
+
+BASH_DIR=$(dirname "${BASH_SOURCE[0]}")
+source "$BASH_DIR"/utils.sh
+
+ray stop --force
+
+
+# DO NOT change unless you fully understand its purpose
+export HABANA_VISIBLE_DEVICES="ALL"
+export PT_HPU_ENABLE_LAZY_COLLECTIVES="true"
+export VLLM_RAY_DISABLE_LOG_TO_DRIVER="1"
+export RAY_IGNORE_UNHANDLED_ERRORS="1"
+export PT_HPU_WEIGHT_SHARING=0
+export HABANA_VISIBLE_MODULES="0,1,2,3,4,5,6,7"
+export PT_HPUGRAPH_DISABLE_TENSOR_CACHE=1
+export PT_HPU_LAZY_MODE=1
+export VLLM_MLA_DISABLE_REQUANTIZATION=1
+
+export VLLM_MOE_N_SLICE=8
+export VLLM_EP_SIZE=8
+
+block_size=128
+
+# memory footprint tuning params
+export VLLM_GPU_MEMORY_UTILIZATION=0.9
+export VLLM_GRAPH_RESERVED_MEM=0.4
+export VLLM_GRAPH_PROMPT_RATIO=0
+export VLLM_DELAYED_SAMPLING="true"
+
+# params
+max_model_len=16384
+max_num_batched_tokens=16384
+max_num_seqs=256
+input_min=1
+input_max=16384
+output_max=16384
+model_path=deepseek-ai/DeepSeek-R1
+
+unset VLLM_PROMPT_BS_BUCKET_MIN VLLM_PROMPT_BS_BUCKET_STEP VLLM_PROMPT_BS_BUCKET_MAX
+unset VLLM_PROMPT_SEQ_BUCKET_MIN VLLM_PROMPT_SEQ_BUCKET_STEP VLLM_PROMPT_SEQ_BUCKET_MAX
+unset VLLM_DECODE_BS_BUCKET_MIN VLLM_DECODE_BS_BUCKET_STEP VLLM_DECODE_BS_BUCKET_MAX
+unset VLLM_DECODE_BLOCK_BUCKET_MIN VLLM_DECODE_BLOCK_BUCKET_STEP VLLM_DECODE_BLOCK_BUCKET_MAX
+
+set_bucketing
+
+python3 -m vllm.entrypoints.openai.api_server --host 0.0.0.0 --port 8688 \
+--block-size 128 \
+--model $model_path \
+--device hpu \
+--dtype bfloat16 \
+--kv-cache-dtype fp8_inc \
```

Comment:
If we added fp8_inc as kv_cache_dtype, that should be only working with INC.
Suggest to provide two scripts : one for launching bf16 serving server, one for launching bf8 serving server;